docs/advanced_reference/commands/autonomy_replay.md

```diff
@@ -0,0 +1,129 @@
+## Usage
+```bash
+autonomy replay [OPTIONS] COMMAND [ARGS]
+```
+
+## Description
+
+Tools for replay agent execution.
+
+This meta-command aggregates a number of functionalities for re-running agents using data dumps from previous runs. See the appropriate subcommands for more information. Note that **replay functionalities only work for deployments which were ran in dev mode.**
+
+
+## Options
+
+`--help`
+:   Show the help message and exit.
+
+## Subcommand `autonomy replay agent`
+
+### Usage
+```bash
+autonomy replay agent [OPTIONS] AGENT
+```
+
+### Description
+
+Re-run agent execution.
+
+### Options
+`--build PATH`
+:   Path to the build folder.
+
+`--registry PATH`
+:   Path to the local registry folder.
+
+`--help`
+:   Show the help message and exit.
+
+## Subcommand `autonomy replay tendermint`
+
+### Usage
+```bash
+autonomy replay tendermint [OPTIONS]
+```
+
+### Description
+Tendermint runner.
+
+### Options
+`--build PATH`
+:   Path to the build folder.
+
+`--help`
+:   Show the help message and exit.
+
+
+## Examples
+
+The example below shows how to replay agent runs from Tendermint dumps.
+
+
+1.  Copy and paste the following Makefile into your workspace folder:
```

Comment:
This is wrong, this example just create a dev setup and uses it for replays later. Please redirect the user to the development setup guide and then explain how to use the replays on the dev setup.

Reply:
Will add a ticket to address in another PR